docs/usage/configuration-options.md

```diff
@@ -750,8 +785,10 @@ For example, if you are following a tag like `next` and then that stream is rele
 You can customize the Git author that's used whenever Renovate creates a commit.
 The `gitAuthor` option accepts a RFC5322-compliant string.
 
-**Note** We strongly recommend that the Git author email you use is unique to Renovate.
-Otherwise, if another bot or human shares the same email and pushes to one of Renovate's branches then Renovate will mistake the branch as unmodified and potentially force push over the changes.
+<!-- prettier-ignore -->
+!!! danger
+    We strongly recommend that the Git author email you use is unique to Renovate.
+    Otherwise, if another bot or human shares the same email and pushes to one of Renovate's branches then Renovate will mistake the branch as unmodified and potentially force push over the changes.
```

Comment:
I think this deserves a `danger` type admonition, as you can risk force pushing over changes in a branch.